src/sbmlutils/report/formating.py

```diff
@@ -169,12 +169,17 @@ def boundsStringFromReaction(reaction: libsbml.Reaction, model: libsbml.Model) -
             if ub_p.isSetValue():
                 ub_value = ub_p.getValue()
         if (lb_value is not None) or (ub_value is not None):
-            bounds = f'<code>[{lb_value} <i class="fa fa-sort fa-rotate-90" aria-hidden="true"></i> {ub_value}]</code>'
+            bounds = f"""
```

Comment:
One of the instances, where I have made the HTML code multi-line to overcome the flake8 error

Reply:
In the first doc string for the files ```xslt/math.py``` (line 2) and ```formating.py```(line 2), flake8 says that they should be single line doc-strings. Should I convert them to single line doc strings or shall Iet them be as they are. If we need to update those doc strings in the future to add more detail, then we might have to convert them to multi-line doc strings. Hence I wanted to take your suggestion on what would be best way to go about them.

Reply:
xlst will be removed soon, so no updates here. Multi-line comment is fine.